### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by PrEvAl85** [misskey-dev/misskey#14215] [Q&A] [answered]
**How to add translation function to misskey?**

**Comment by tai-cha** [comment]
> Misskey is available in many languages (also called i18n, which is an abbreviation for Internationalization). While the original language is generally japanese, volunteers are translating Misskey into other languages. Misskey uses a service called Crowdin to manage its translations.

https://crowdin.com/project/misskey

quoted from https://misskey-hub.net/en/docs/about-misskey/

**Comment by Sayamame-beans** [comment]
If you mean note translation function, you need to prepare DeepL Auth Key.
https://github.com/misskey-dev/misskey/blob/ce39c3a2fbf9ab9efaff82fdbad84ac9f2311c6e/CHANGELOG.md?plain=1#L2495-L2496

**Reply by PrEvAl85** [reply]
But is it a paid feature? Is there any free plugin available?

**Reply by KisaragiEffective** [reply, marked as answer]
> But is it a paid feature?

No, there is free tier. You can translate 500,000 characters per month ([deepl.com](https://www.deepl.com/en/pro-api)) via their REST API without paying.
